BAZProjectC1/Extensions/UIImageViewExtension.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  UIImageViewExtension.swift
+//  BAZProjectC1
+//
+//  Created by 1044336 on 17/09/22.
+//
+
+import Foundation
+import UIKit
+extension UIImageView{
```

Comment:
Nitpick: Agrega un salto de línea entre los imports y la definición de tu extensión
Deja un espacio entre el nombre de la clase y las llaves
```suggestion
import UIKit

extension UIImageView {
```